src/Umbraco.Web.UI.Client/src/views/propertyeditors/blocklist/umbblocklistblock.component.js

```diff
@@ -48,8 +48,7 @@
             $scope.valFormManager = model.valFormManager;
 
             if (model.stylesheet) {
-                // TODO: Not sure why this needs a prefixed /? this means it will never work in a virtual directory
-                model.stylesheet = "/" + model.stylesheet;
+                model.stylesheet = umbRequestHelper.ensureVirtualPath(model.stylesheet);
```

Comment:
I assume somewhere along the way that `model.stylesheet` then gets converted with umbRequestHelper.convertVirtualToAbsolutePath ?

Reply:
No this got missed - good catches will fix up